tests/test_footprint1.py

```diff
@@ -1016,6 +1016,27 @@ def testEdge(self):
         self.checkEdge(afwDetect.Footprint.readFits(
             os.path.join(testPath, "testFootprintEdge.fits")))
 
+    def testExtractImage(self):
+        """Test image extraction from a Footprint"""
+        image = afwImage.Image(
+            np.arange(100, dtype=np.int32).reshape(10, 10),
+            xy0=lsst.geom.Point2I(23, 25),
+            dtype="I"
+        )
+        radius = 3
+        spans = afwGeom.SpanSet.fromShape(radius, afwGeom.Stencil.CIRCLE, offset=(27, 30))
+        footprint = afwDetect.Footprint(spans)
+
+        # The extracted footprint should be the same as the product of the
+        # spans and the overlapped bow with the image
+        truth = spans.asArray() * image.array[2:9, 1:8]
+        self.assertEqual(footprint.extractFluxFromImage(image), np.sum(truth))
+        # Test hard coded value
+        self.assertEqual(footprint.extractFluxFromImage(image), 1566)
+
+        # Test the array method with an offset
+        self.assertEqual(footprint.extractFluxFromArray(image.array-3, image.getBBox().getMin()), 1566-29*3)
```

Comment:
Why 29*3? Probably worth a comment, since it's not obvious to me why that's the right value for this offset.